package.json

```diff
@@ -33,6 +33,6 @@
     "remark-slug": "^4.2.3"
   },
   "engines": {
-    "node": ">=7"
+    "node": ">=10"
```

Comment:
Node 8 is still supported, so I think we continue to allow it here unless there's a specific reason not to.

See https://github.com/nodejs/Release#release-schedule

Reply:
Then let me test locally if all the deps work without problems on node v8.